support-frontend/assets/components/priceCards/priceCardsContainer.tsx

```diff
@@ -33,43 +42,67 @@ const contributionTypeToPaymentInterval: Partial<
 };
 
 export function PriceCardsContainer({
-	frequency,
+	paymentFrequency,
 	renderPriceCards,
 }: PriceCardsContainerProps): JSX.Element {
 	const dispatch = useContributionsDispatch();
-	const currency = useContributionsSelector(
-		(state) => state.common.internationalisation.currencyId,
+	const { countryGroupId, currencyId } = useContributionsSelector(
+		(state) => state.common.internationalisation,
 	);
 	const { amounts } = useContributionsSelector((state) => state.common);
 	const { amountsCardData } = amounts;
 	const { selectedAmounts, otherAmounts } = useContributionsSelector(
 		(state) => state.page.checkoutForm.product,
 	);
 	const minAmount = useContributionsSelector(getMinimumContributionAmount());
+
+	const inThreeTierVariantB = inThreeTierV2VariantB(
+		useContributionsSelector((state) => state.common).abParticipations,
+	);
+	const tierCards = inThreeTierVariantB ? tierCardsVariantB : tierCardsVariantA;
+
+	const getTierLowPriceCardAmounts = (
+		contributionType: ContributionType,
+		countryGroup: CountryGroupId,
+		intThreeTierVariant: boolean,
+	): AmountValuesObject | undefined => {
+		const tierBillingPeriod =
+			contributionType === 'ANNUAL' ? 'annual' : 'monthly';
+		const priceCards = tierCards[`tier1`].plans[tierBillingPeriod].priceCards;
+		if (priceCards && intThreeTierVariant) {
+			return priceCards[countryGroup];
```

Comment:
This function only returns an `AmountValuesObject` if `intThreeTierVariant` is `true` - otherwise it returns `undefined`. Rather than making this check at the end of the function we could simplify it by checking whether `intThreeTierVariant` is `true` at the very top of the and just returning immediately if it's `false`? I think it might not even be necessary to have this function

Instead of...

```
	const {
		amounts: frequencyAmounts,
		defaultAmount,
		hideChooseYourAmount,
	} = getTierLowPriceCardAmounts(
		paymentFrequency,
		countryGroupId,
		inThreeTierVariantB,
	) ?? amountsCardData[paymentFrequency];
```
Could we do...

```
const {
		amounts: frequencyAmounts,
		defaultAmount,
		hideChooseYourAmount,
	} = inThreeTierVariantB ? tierCardsVariantB[`tier1`].plans[tierBillingPeriod].priceCards[countryGroup] : amountsCardData[paymentFrequency];
```

We could then get rid of `getTierLowPriceCardAmounts` and the `tierCards` variable referenced in my comment above?

Reply:
Using your suggestion above,  I have simplified and remove `getTierLowPricaeCardAmounts` as requetsed with this rewrite:`	const tierBillingPeriod =
		paymentFrequency === 'ANNUAL' ? 'annual' : 'monthly';
	const tierCardData =
		tierCardsVariantB.tier1.plans[tierBillingPeriod].priceCards;
	const {
		amounts: frequencyAmounts,
		defaultAmount,
		hideChooseYourAmount,
	} = tierCardData
		? tierCardData[countryGroupId]
		: amountsCardData[paymentFrequency];`

Reply:
updated as shown above

Reply:
Looks good @paul-daniel-dempsey 👍